IFComp/lib/IFComp/Form/Entry.pm

```diff
@@ -27,6 +27,66 @@ has_field 'subtitle' => (
     maxlength => 128,
 );
 
+has_field 'platform' => (
+    type         => 'Select',
+    id           => 'platform',
+    label        => 'Gameplay platform',
+    options      => [
+        {
+            value => 'website',
+            label => 'Browser-based JavaScript (including Twine)',
+        },
+        {
+            value => 'inform',
+            label => 'Glulx or Z-code, not packaged with a website',
+        },
+        {
+            value => 'quixe',
+            label => q{Glulx, using Inform's "Release with a website" feature},
+        },
+        {
+            value => 'parchment',
+            label => q{Z-code, using Inform's "Release with a website" feature},
+        },
+        {
+            value => 'inform-website',
```

Comment:
What's the value of having these three Inform website options here? How will they be handled differently?

Reply:
It's mostly for labeling purposes on the ballot.